.github/workflows/ci.yml

```diff
@@ -373,7 +373,7 @@ jobs:
             ./gradlew app:assemblePR
           popd
           mkdir -p artifacts
-          mv src/openrct2-android/app/build/outputs/apk/arm/pr/app-arm-pr.apk artifacts/openrct2-arm.apk
+          cp src/openrct2-android/app/build/outputs/apk/arm/pr/app-arm-pr.apk artifacts/openrct2-arm.apk
```

Comment:
Why?

Reply:
Still waiting on the reason for this change, otherwise I think the PR looks fine.

Reply:
It's to "preserve" the original file, but I can undo it.

Reply:
It's already cleaned up soon after this, isn't it? It's moved to `artifacts` which will then be used to upload and then the action finishes. `mv` should technically be faster